homeassistant/components/braviatv/media_player.py

```diff
@@ -253,29 +146,31 @@ def device_info(self):
     @property
     def state(self):
         """Return the state of the device."""
-        return self._state
+        if self._client.is_on:
```

Comment:
Please change all of these. 
```suggestion
        if self.coordinator.is_on:
```